portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,67 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+h1 {
+  text-align: center;
+}
+
+body {
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  text-align: justify;
+}
+
+.content-container {
+  max-width: 650px;
+}
+
+div {
+  width: 800px;
+  border: 25px solid;
+  border-color: rgba(50, 100, 230, 0.0);
```

Comment:
Use the border shorthand syntax to do this in one line.

https://developer.mozilla.org/en-US/docs/Web/CSS/border

Reply:
Thanks for the pointer. I didn't realize that putting the color on the same line would still work. I actually had that there while I was debugging, which is why it's transparent. I got rid of it.